app/Http/Controllers/PostsController.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Post;
+use App\PostStatus;
+use App\User;
+use Illuminate\Http\Request;
+
+class PostsController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $posts = Post::all();
+
+        return view('posts.index', ['posts' => $posts]);
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        $users                 = User::all();
+        $post_statuses         = PostStatus::all();
```

Comment:
All variable names within php-code in most cases should follow camelCase format

Reply:
Done